static/src/javascripts/bootstraps/standalone.commercial.ts

```diff
@@ -38,20 +38,8 @@ const tags: Record<string, unknown> = {
 	bundle: 'standalone',
 };
 
-// modules necessary to load an ad
-const commercialBaseModules: Modules = [
-	['cm-setAdTestCookie', setAdTestCookie],
-	['cm-prepare-prebid', preparePrebid],
-	// Permutive init code must run before google tag enableServices()
-	// The permutive lib however is loaded async with the third party tags
-	['cm-prepare-googletag', () => initPermutive().then(prepareGoogletag)],
-	['cm-prepare-a9', prepareA9],
-	['cm-mobileSticky', initMobileSticky],
-	['cm-highMerch', initHighMerch],
-	['cm-articleAsideAdverts', initArticleAsideAdverts],
-	['cm-articleBodyAdverts', initArticleBodyAdverts],
-	['cm-liveblogAdverts', initLiveblogAdverts],
-];
+// modules necessary to load the first ads on the page
+const commercialBaseModules: Modules = [];
```

Comment:
Thinking about this a bit more… could we do a ternary? And use JSDoc to keep the comment attached to the variable?

```ts
/** Base modules in the critical path for loading an ad.
 * If ad-free, this will be an empty array.
 */
const commercialBaseModules: Modules = commercialFeatures.adFree ? [] : [ /* … */ ]
```